cartographer/mapping_2d/sparse_pose_graph.cc

```diff
@@ -294,30 +294,25 @@ void SparsePoseGraph::WaitForAllComputations() {
   common::MutexLocker locker(&mutex_);
   const int num_finished_scans_at_start =
       constraint_builder_.GetNumFinishedScans();
-  while (!locker.AwaitWithTimeout(
-      [this]() REQUIRES(mutex_) {
-        return static_cast<size_t>(constraint_builder_.GetNumFinishedScans()) ==
-               trajectory_nodes_.size();
-      },
-      common::FromSeconds(1.))) {
+  while (!locker.AwaitWithTimeout([this]() REQUIRES(mutex_) {
```

Comment:
Correct, I have a particularly aggressive editor configuration.

@jihoonl I'll have the `cartographer_ros` PR ready to go before they merge this one, but I haven't started on it yet.